src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
```

Comment:
@KirkMunro, your last commit had 9 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/engine/ExperimentalFeature/Get-ExperimentalFeature.Tests.ps1: line 176
176:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```
ForEach-Object -Parallel common parameters.Verifies that ErrorAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:128 char:40
    + ... k = { 1..1 | ForEach-Object -Parallel { "Hello" } -ErrorAction Stop }
    +                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that WarningAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:132 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -WarningAction Silen ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that InformationAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:136 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -InformationAction S ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that PipelineVariable common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:140 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -PipelineVariable pi ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```